src/libraries/Microsoft.PowerFx.Core/Public/ErrorKind.cs

```diff
@@ -33,8 +33,6 @@ public enum ErrorKind
         InsufficientMemory = 21,
         QuotaExceeded = 22,
         Network = 23,
-        Numeric = 24,
-        InvalidArgument = 25,
```

Comment:
Maybe keep InvalidArgument? Looks like that was just fixing something that was missing

Reply:
Good point - added it back